.github/pyinstaller/pyinstaller.spec

```diff
@@ -1,10 +1,40 @@
 # -*- mode: python -*-
 # Copyright (C) 2020 Mandiant, Inc. All Rights Reserved.
-import os.path
-import subprocess
+import sys
+import logging
 
 import wcwidth
 
+from pathlib import Path
+
+logger = logging.getLogger(__name__)
+
+def generate_rule_cache(capa_dir: Path):
```

Comment:
if we want to keep the cache-ruleset.py script around, let's create this function there and call from here?

Reply:
I thought of this as well, but how do you think we can import `scripts/cache-ruleset.py` given it has a hyphen in the filename? Discussion [here](https://stackoverflow.com/questions/761519/is-it-ok-to-use-dashes-in-python-files-when-trying-to-import-them).

Reply:
things in "scripts" shouldn't be reusable library code imho. So we could find a common place to refactor the logic into. Maybe `capa/rules/cache.py`?

Reply:
oh yeah, moving it to a proper place is much better